include/aos/common/pkcs11/pkcs11.hpp

```diff
@@ -456,24 +456,76 @@ class SessionContext {
     CK_FUNCTION_LIST_PTR mFunctionList;
 };
 
+#if AOS_CONFIG_PKCS11_USE_STATIC_LIB == 1
```

Comment:
`#if AOS_CONFIG_PKCS11_USE_STATIC_LIB` would be enough for bool type defines.

Reply:
done